resources/lib/services/msl/msl_handler.py

```diff
@@ -26,9 +26,14 @@
 
 CHROME_BASE_URL = 'https://www.netflix.com/nq/msl_v1/cadmium/'
 ENDPOINTS = {
+<<<<<<< HEAD
```

Comment:
This file also has unresolved merge conflicts

Reply:
fixed